### PR TITLE
feature: add support for multiple elements highlight

### PR DESCRIPTION
**Pull request by IArny**
I would propose feature to support multiple elements highlight. All elements selected with `attachTo.element ` will be highlighted if `attachTo.multiple` is `true`. I added `multiple` flag to `attachTo` field to keep old behavior. I'd appreciate your feedback for this PR.

### CODE REVIEW

**Comment by IArny** [comment]
Also this PR can resolve https://github.com/shipshapecode/shepherd/issues/525 issue

**Comment by IArny** [comment]
@rwwagner90  could you review this PR please

**Comment by IArny** [comment]
@monshan Could you find a time for this PR?

File: index.html [hunk 1]
```diff
@@ -271,7 +271,7 @@ <h3 class="demo-heading font-heading text-2xl uppercase">
           </div>
 
           <div class="hero-followup font-heading mb-12 mt-12 lg:mb-24 lg:mt-24">
-            <div class="bg-navy inline-block mb-4 w-56 lg:mr-4">
+            <div class=" bg-navy inline-block mb-4 w-56 lg:mr-4">
```
Comment (rwwagner90) [inline index.html:274]:
```suggestion
            <div class="bg-navy inline-block mb-4 w-56 lg:mr-4">
```

File: landing/js/welcome.js [hunk 2]
```diff
@@ -125,6 +125,31 @@
           }
         ],
         id: 'attaching'
+      },
+	  {
```
Comment (rwwagner90) [inline landing/js/welcome.js:129]:
```suggestion
    {
```

File: landing/js/welcome.js [hunk 3]
```diff
@@ -125,6 +125,31 @@
...
+        title: 'Highliting multiple Elements',
+        text: 'Your tour steps can highlight multiple elements (like this step).',
+        attachTo: {
+          element: '.hero-multiple-select',
+		  on: 'bottom',
```
Comment (rwwagner90) [inline landing/js/welcome.js:134]: Lots of weird spacing issues in here. Could you please auto format this with prettier?

File: src/js/components/shepherd-modal.svelte [hunk 4]
```diff
@@ -39,26 +36,32 @@
...
-      openingProperties = {
-        width: width + modalOverlayOpeningPadding * 2,
-        height: height + modalOverlayOpeningPadding * 2,
-        x: (x || left) - modalOverlayOpeningPadding,
-        y: y - modalOverlayOpeningPadding,
-        r: modalOverlayOpeningRadius
-      };
+	if (targetElements) {
+	  const elements = [];
+      targetElements.forEach(el => {
```
Comment (rwwagner90) [inline src/js/components/shepherd-modal.svelte:49]: Spacing issues here as well

**Review by rwwagner90 — CHANGES_REQUESTED** [review]

**Comment by rwwagner90** [comment]
Thanks for the PR @IArny! Lots of formatting issues with the spacing in this code, please run prettier and clean things up and I will take another look. I think in general we would love to support this feature, but I am not sure if this is the best API for it.

**Comment by IArny** [comment]
@rwwagner90 thank you for your response. I closed this PR by mistake, but  already opened the new one https://github.com/shipshapecode/shepherd/pull/2083 . I ran prettier and it fixed a little bit more files than were changed by me, hope it's fine.

**Pull request closed without merging**